common/types/wire/doc.go

```diff
@@ -0,0 +1,6 @@
+/*
+Package wire contains types that are encoded for:
+- sending over the network
+- signing
+*/
```

Comment:
This isn't easily achievable. The packages containing the `Handlers` and `Builders` should contain their respective wire types in them. ATXs (or their wire types) should be in the `activation` package. Proposal wire types should be in the `proposals` package, etc.

`common` should be reserved for domain objects, i.e. the types used by the services but not used for encoding.